### PR TITLE
feat: insert identities

### PR DESCRIPTION
**Pull request by QuantumExplorer**
## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: node/test/Drive.spec.js [hunk 1]
```diff
@@ -211,6 +227,17 @@ describe('Drive', () => {
     });
   });
 
+  describe('#insertIdentities', () => {
```
Comment (shumkov) [inline node/test/Drive.spec.js]:
```suggestion
  describe('#insertIdentity', () => {
```

File: drive/src/contract/types.rs [hunk 2]
```diff
@@ -155,7 +177,18 @@ impl DocumentFieldType {
...
+                    .map(|(string, field_type)| {
+                        (
+                            Value::Text(string.clone()),
+                            field_type.random_filled_value(rng),
+                        )
+                    })
+                    .collect();
+                Value::Map(value_vec)
+            }
             DocumentFieldType::Array => Value::Null,
```
Comment (shumkov) [inline drive/src/contract/types.rs:192]: We could put a lot of data here and it won't be taken into account.

Comment (QuantumExplorer) [reply drive/src/contract/types.rs:192]: I will fix it in another PR focused on array support okay?

Comment (shumkov) [reply drive/src/contract/types.rs:192]: Sounds good

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**